style.ts

```diff
@@ -5,6 +5,12 @@ import {
   GeoStylerStringFunction
 } from './functions';
 
+export interface LikeAttributes {
+  wildCard: string;
+  singleChar: string;
+  escape: string;
+}
```

Comment:
I think an array like structure would fit the geostyler-style more. e.g.:
```suggestion
/**
 * The LikeAttributes describe the behaviour of the {@link LikeFilter}.
 * [
 *    wildcard, (default should be `*`),
 *    singleChar, (default should be `.`),
 *    escape, (default should be `!`),
 *  ]
 */
export interface LikeAttributes [string, string, string];
```

... this was just written freestyle on github so its just for orientation. :)